modules/core/include/visp3/core/vpImageTools.h

```diff
@@ -796,6 +799,28 @@ void vpImageTools::undistort(const vpImage<Type> &I, const vpCameraParameters &c
 #endif
 }
 
+/*!
+  Undistort an image distorted by fisheye lens
+
+  \param I : Input image to undistort.
+
+  \param cam : Parameters of the camera causing distortion.
+
+  \param undistI : Undistorted output image. The size of this image
+  will be the same than the input image \e I.
+
+  \sa initUndistortMap, remap
+*/
+template<class Type>
+void vpImageTools::undistortFisheye(const vpImage<Type> &I, const vpCameraParameters &cam, vpImage<Type> &undistI)
```

Comment:
There is [`undistort()`](https://visp-doc.inria.fr/doxygen/visp-3.3.0/classvpImageTools.html#aed4d783a3c6e85521b1903d3322eb8bc).

I would suggest to modify `undistort()` instead.

Reply:
Yes